24.md

```diff
@@ -0,0 +1,57 @@
+NIP-24
+======
+
+Protect the Unknown
+-------------------
+
+`draft` `optional` `client` `author:giszmo`
+
+## Replaceable Events
+
+A *replaceable event* as defined in [NIP-01](01.md) and [NIP-16](16.md).
```

Comment:
are kind 0 and kind 3 considered a replaceable event? If so should NIP-16 be amended to state that? If not, should we explicitly state these are also included in this NIP?

it seems that this does cover kind 0 and kind 3, but the definition in NIP-16 of replaceable events doesn't include these.

Reply:
kind 0 and 3 are covered by nip-01 mentioned above

Reply:
yes but nip-01 does not define them as a replaceable event. was pretty confusing to me but just my opinion. i know kind 0 and kind 3 are replaceable, but nip-16 explicitly defines what we consider to be a "replaceable event". if we want to use terms like this we should be clear on the definition and not use it in different contexts in different nips.